src/Slider/Slider.tsx

```diff
@@ -111,29 +125,31 @@ const Slider: React.FunctionComponent<SliderProps> = (props: SliderProps): React
     function getActiveTrackStyles(): React.CSSProperties {
         const calculatedThumbPosition: number = getPercentage();
         let zeroPosition: number;
+        const appearance: SliderAppearance = props.appearance || defaultAppearance;
+        const { width, offset }: AppearanceStyleMap[keyof AppearanceStyleMap] = appearanceSizesMap[appearance];
         const style: React.CSSProperties = {};
         if (min >= 0) {
             zeroPosition = 0;
             style.left = `${zeroPosition}%`;
-            style.width = `calc(${calculatedThumbPosition}% + 27px)`;
+            style.width = `calc(${calculatedThumbPosition}% + ${width})`;
         } else if (max <= 0) {
             zeroPosition = 100;
-            style.left = `calc(${zeroPosition}% + 56px)`;
-            style.width = `calc(${100 - calculatedThumbPosition}% + 27px)`;
+            style.left = `calc(${zeroPosition}% + ${offset})`;
+            style.width = `calc(${100 - calculatedThumbPosition}% + ${width})`;
             style.transform = "rotateY(180deg)";
         } else {
             if (props.value <= 0) {
                 zeroPosition = size ? Math.abs((min / size) * 100) : 0;
-                style.left = `calc(${zeroPosition}% + 27px)`;
+                style.left = `calc(${zeroPosition}% + ${width})`;
                 style.width = zeroPosition - calculatedThumbPosition + "%";
                 style.transform = "rotateY(180deg)";
             } else {
                 zeroPosition = size ? Math.abs(100 - (max / size) * 100) : 0;
-                style.left = `calc(${zeroPosition}% + 27px)`;
+                style.left = `calc(${zeroPosition}% + ${width})`;
                 style.width = calculatedThumbPosition - zeroPosition + "%";
             }
         }
-        return style;
+        return { ...style };
```

Comment:
This is unnecessary processing, you don't need to spread something that you just created.

Reply:
good catch, you're right